### PR TITLE
FeedbackViewController partial modal

### PR DESCRIPTION
**Pull request by JThramer**
This is the PR for the partial Feedback Controller modal, now with 100% less Frankenstein! Shout-out to @ericrwolfe for working his git magic.

/cc @frederoni @bsudekum

### CODE REVIEW

File: MapboxNavigation/FeedbackViewController.swift [hunk 1]
```diff
@@ -2,31 +2,43 @@ import UIKit
...
     typealias SendFeedbackHandler = (FeedbackItem) -> ()
     
     var sendFeedbackHandler: SendFeedbackHandler?
     var dismissFeedbackHandler: (() -> ())?
+    var sections = [FeedbackSection]()
+    
+    let cellReuseIdentifier = "collectionViewCellId"
+    let interactor = Interactor()
+    
+    let autodismissInterval: TimeInterval = 5
```
Comment (bsudekum) [inline MapboxNavigation/FeedbackViewController.swift]: Nit: autoDismissInterval

**Comment by bsudekum** [comment]
Looks good. One improvement that would make the UX a little better is that if the faded background that is used for the feedback UI is also the faded background for "Thank you for your report!". Right now it goes from faded -> not faded -> faded and it's a bit jarring.

**Comment by frederoni** [comment]
@bsudekum this PR is ready for review

**Comment by frederoni** [comment]
@bsudekum removed the dimming view behind the confirmation dialog

**Pull request merged**